com.github.hugolabe.Wike.json

```diff
@@ -0,0 +1,44 @@
+{
+  "app-id" : "com.github.hugolabe.Wike",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "3.38",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "wike",
+  "finish-args" : [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio"
```

Comment:
I suppose this is used to play audio/video in a wiki article?

Reply:
I have put it because the application emits a beep sound when the text search does not give any results.